src/dmd/backend/outbuf.d

```diff
@@ -294,40 +260,6 @@ struct Outbuffer
         writeString(cast(const(char)[])(s));
     }
 
```

Comment:
prependBytes, prepend, bracket are use by other projects that use the backend.

Reply:
Bit bummed about this. It means we can't know for sure where the unused code is. "If it's not under source control it doesn't exist" and all that.

I'll re-add the code.

Reply:
This is the issue I mentioned at the last Fondation meeting, and why I'd like to see https://github.com/DigitalMars/Compiler/pull/12 done one day.